sky/resources.py

```diff
@@ -799,15 +800,17 @@ def _try_validate_image_id(self) -> None:
                     'Cloud must be specified when image_id is provided.')
 
         # Apr, 2023 by Hysun(hysun.he@oracle.com): Added support for OCI
```

Comment:
This line can probably be removed now : )

Reply:
Good point! Removed. Thanks!